OpenOversight/tests/test_functional.py

```diff
@@ -0,0 +1,29 @@
+import time
+
+
+def test_user_can_load_homepage_and_get_to_form(mockdata, browser):
+    browser.get("http://localhost:5000")
+
+    # Complainant loads homepage
+    assert "OpenOversight" in browser.title
+    browser.find_element_by_link_text("Try it").click()
```

Comment:
It's best to assign an id to that button (in https://github.com/lucyparsons/OpenOversight/blob/develop/OpenOversight/app/templates/index.html#L9) and then find the element by id. This makes it less fragile and in a world where we have i18n concerns (which, in a multicultural city like Chicago should be p. soon) we wouldn't need different integration tests for English, Spanish, Polish, Arabic, Tagalog, Chinese, etc